pkg/cli/cmd/app/graph/graph.go

```diff
@@ -24,11 +24,10 @@ import (
 func NewCommand(factory framework.Factory) (*cobra.Command, framework.Runner) {
 	runner := NewRunner(factory)
 	cmd := &cobra.Command{
-		Use:     "graph",
-		Aliases: []string{"connections"},
-		Short:   "Shows the application graph for an application.",
-		Long:    `Shows the application graph for an application.`,
-		Args:    cobra.MaximumNArgs(1),
+		Use:   "graph",
```

Comment:
I think cli wrapper in functional test still uses `connections`. So please update them too.

Reply:
I think they are cli/connections, I checked for app/connections but could not find any. Did  I miss something?

Reply:
I was thinking that functional test was failing because it might use connections. I saw that functional tests are passing now. I will approve this PR.